src/App/App.test.js

```diff
@@ -1,8 +1,98 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
-
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
+import React from 'react'
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom'
+import App from './App.js'
+import { callApi, callSingleApi } from '../callApis';
+jest.mock('../callApis.js');
+
+  beforeEach(() => {
+
+    callApi.mockResolvedValue({
+      movies: [
+        {
+          "id": 694919,
+          "poster_path": "https://image.tmdb.org/t/p/original//6CoRTJTmijhBLJTUNoVSUNxZMEI.jpg",
+          "backdrop_path": "https://image.tmdb.org/t/p/original//pq0JSpwyT2URytdFG0euztQPAyR.jpg",
+          "title": "Money Plane",
+          "average_rating": 6.666666666666667,
+          "release_date": "2020-09-29"
+        },
+        {
+          "id": 337401,
+          "poster_path": "https://image.tmdb.org/t/p/original//aKx1ARwG55zZ0GpRvU2WrGrCG9o.jpg",
+          "backdrop_path": "https://image.tmdb.org/t/p/original//zzWGRw277MNoCs3zhyG3YmYQsXv.jpg",
+          "title": "Mulan",
+          "average_rating": 4.909090909090909,
+          "release_date": "2020-09-04"
+        }
+      ]
```

Comment:
Consider storing your mock objects in a separate file, and then pulling them into this test file. It will make your test suite easier to read.